pkg/verify/certificate_identity.go

```diff
@@ -116,15 +117,19 @@ func (s SubjectAlternativeNameMatcher) Verify(actualCert certificate.Summary) er
 	return nil
 }
 
-func NewCertificateIdentity(sanMatcher SubjectAlternativeNameMatcher, extensions certificate.Extensions) (CertificateIdentity, error) {
+func NewCertificateIdentity(sanMatcher SubjectAlternativeNameMatcher, issuerRegexp *regexp.Regexp, extensions certificate.Extensions) (CertificateIdentity, error) {
```

Comment:
It might be more consistent with `SubjectAlternativeNameMatcher.Regexp` to not use a pointer here.

Reply:
Makes sense; this got replaced with `IssuerMatcher`